x-pack/plugins/stack_connectors/public/connector_types/microsoft_defender_endpoint/microsoft_defender_endpoint_connector.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  ActionConnectorFieldsProps,
+  ConfigFieldSchema,
+  SecretsFieldSchema,
+  SimpleConnectorForm,
+} from '@kbn/triggers-actions-ui-plugin/public';
+import * as translations from './translations';
+
+const configFormSchema: ConfigFieldSchema[] = [
+  {
+    id: 'clientId',
+    label: translations.CLIENT_ID_LABEL,
+    isRequired: true,
+  },
+  {
+    id: 'tenantId',
+    label: translations.TENANT_ID_LABEL,
+    isRequired: true,
+  },
+  {
+    id: 'oAuthServerUrl',
+    label: translations.OAUTH_URL_LABEL,
+    isRequired: true,
+    isUrlField: true,
+    defaultValue: 'https://login.microsoftonline.com',
+  },
+  {
+    id: 'oAuthScope',
+    label: translations.OAUTH_SCOPE,
+    isRequired: true,
+    defaultValue: 'https://securitycenter.onmicrosoft.com/windowsatpservice/.default',
```

Comment:
Is `onmicrosoft` on purpose?

Reply:
Ok just to confirm, there are two urls: `https://securitycenter.onmicrosoft.com/` and `https://securitycenter.microsoft.com/` that we're using for this connector?

Reply:
Correct.

Scope - although it may look like a URL - it is used in the MS OAuth flow to identify where privileges are coming from.  I been searching where I saw this documented for a bit now and can't find it. I think it was provided to me when I was setting up the App in azure.

Reply:
Sounds good! Thanks :)